testing/python/transform/test_tilelang_transform_legalize_safe_memory_access.py

```
@@ -38,9 +38,93 @@ def assert_vectorize_access(M: int = 64, N: int = 64):
+        with T.Kernel(1, 1, threads=M) as (bx, by):
+            A_shared = T.alloc_shared((M, N), dtype=dtype)
+            tid = T.get_thread_binding()
+
+            T.reads(A[tid + M_offset, N_offset:N + N_offset])
+            for j in T.serial(N):
+                A_shared[tid, j] = T.if_then_else(
+                    j + N_offset < N,
+                    T.if_then_else(tid + M_offset < M, A[tid + M_offset, j + N_offset],
+                                   T.float32(0)), T.float32(0))
+                # Nest if-then-else is expected, do not flatten it to pass structural equal check
+                if j + 2 < 64:  # noqa: SIM102
```

Ruff (0.14.0)

Unused `noqa` directive (non-enabled: `SIM102`)

Remove unused `noqa` directive

(RUF100)